django_evaluation/settings/local.py

```diff
@@ -326,6 +328,24 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
             {"name": title, "url": reverse_lazy(url), "html_id": html_id}
         )
 
+OIDC_RP_CLIENT_ID = os.environ.get("OIDC_RP_CLIENT_ID", "freva")
+OIDC_RP_CLIENT_SECRET = ""
+OIDC_OP_AUTHORIZATION_ENDPOINT = OIDC_URL + "/protocol/openid-connect/auth"
+OIDC_OP_TOKEN_ENDPOINT = OIDC_URL + "/protocol/openid-connect/token"
+OIDC_OP_USER_ENDPOINT = OIDC_URL + "/protocol/openid-connect/userinfo"
+OIDC_OP_JWKS_ENDPOINT = OIDC_URL + "/protocol/openid-connect/certs"
+OIDC_RP_SIGN_ALGO = "RS256"
+OIDC_CREATE_USER = True
+OIDC_STORE_ID_TOKEN = True  # important for the logout callback
+OIDC_STORE_ACCESS_TOKEN = True  # important for the logout callback
+
```

Comment:
You should get those from the response of the discovery url. Check out freva-rest